src/OSPSuite.Core/Domain/ApplicationParameters.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace OSPSuite.Core.Domain
+{
+   public class ApplicationParameters
```

Comment:
This is what we will return. StartTime may not be null but rest might be.
Obviously, application create by user that do not fit our schema won't be found here. But unlikely